### PR TITLE
Allow changing parameters for TEL curve

### PR DESCRIPTION
**Pull request by eslickj**
This let's you change the TEL curve for the steam turbine.  This will change with turbine size, but we were stuck with a fixed curve. 

### Legal Acknowledgement

By contributing to this software project, I agree to the following terms and conditions for my contribution:

1. I agree my contributions are submitted under the license terms described in the LICENSE.txt file at the top level of this directory.
2. I represent I am authorized to make the contributions and grant the license. If my employer has rights to intellectual property that includes these contributions, I represent that I have received permission to make contributions and grant the required license on behalf of that employer.

### CODE REVIEW

File: idaes/power_generation/unit_models/helm/turbine_outlet.py [hunk 1]
```diff
@@ -60,16 +60,29 @@ def build(self):
         self.flow_coeff.fix()
         self.efficiency_mech.fix()
 
+
+        self.tel_c0 = Var(initialize=0.0064)
```
Comment (andrewlee94) [inline idaes/power_generation/unit_models/helm/turbine_outlet.py:64]: Minor nitpicks, but a doc string and explicit units (`units=pyunits.dimensionless`) would be nice.

**Review by andrewlee94 — DISMISSED** [review]
Looks fine - my only comment is a minor thing that doesn't need to be addressed.

**Review by MAZamarripa — DISMISSED** [review]
lgtm

File: idaes/power_generation/unit_models/helm/turbine_outlet.py (hunk 1 above)
Comment (eslickj) [reply idaes/power_generation/unit_models/helm/turbine_outlet.py]: Thanks.  I was kind of lazy with the units there.  I moved the units to the coefficients where they belong, and added doc strings.  @andrewlee94 and @MAZamarripa should be okay to re-review now.

**Review by andrewlee94 — APPROVED** [review]

**Review by MAZamarripa — APPROVED** [review]
lgtm

**Pull request merged**